student/src/test/java/tds/student/sql/repository/remote/RemoteExamItemResponseRepositoryTest.java

```diff
@@ -38,7 +49,79 @@ public void shouldMarkForReviewSuccessful() throws ReturnStatusException {
         final int position = 3;
         final boolean mark = true;
 
+        HttpHeaders headers = new HttpHeaders();
+        headers.set("Accept", MediaType.APPLICATION_JSON_VALUE);
+        headers.setContentType(MediaType.APPLICATION_JSON);
+        HttpEntity<?> requestHttpEntity = new HttpEntity<>(mark, headers);
+
+        URI uri = UriComponentsBuilder.fromHttpUrl(String.format("http://localhost:8080/exam/%s/item/%d/review",
+          opportunityInstance.getExamId(), 3))
+          .queryParam("sessionId", opportunityInstance.getSessionKey())
+          .queryParam("browserId", opportunityInstance.getExamBrowserKey())
+          .build()
+          .toUri();
+
         remoteExamItemResponseRepository.markItemForReview(opportunityInstance, position, mark);
-        verify(mockRestTemplate).exchange(isA(URI.class), isA(HttpMethod.class), isA(HttpEntity.class), isA(ParameterizedTypeReference.class));
```

Comment:
This test was just testing that the restTemplate was called.  Fixed it to actually verify the call has the right data.